watertap/property_models/multicomp_aq_sol_prop_pack.py

```diff
@@ -382,16 +382,24 @@ def build(self):
             else:
                 self.neutral_set.add(j)
 
+        if len(self.neutral_set)>0 and not len(self.ion_set):
+            _log.warning("Since the charge argument was not provided for any solutes while instantiating the MCAS property model, all solutes are assumed to be neutral and have no charge. If any ions exist in solute_list, make sure to provide charge data to avoid erroneous property calculations.")
```

Comment:
This warning will always show up if no charge data is provided, which isn't ideal, but in case this was unintentionally done by the user, I wanted to notify them in some way. I am open to suggestions here.

Overall, if the user mistakenly excludes charge data while instantiating MCAS, they can end up with erroneous calculations.